scripts/get-gee-build-environment.sh

```diff
@@ -0,0 +1,432 @@
+#! /bin/bash
+
+# This scripts installs packages, downloads and buils software used to
+# download the Open GEE source code and build it.
+#   It runs on RHEL 6, RHEL 7, Cent OS 6, Cent OS 7,
+# Ubuntu 14.04, Ubuntu 16.04.
+#   This script does not share code with other shell scripts so that it can be
+# distributed as a single file that can be used to obtain a build environment
```

Comment:
Comment out of date

Reply:
Re-wrote comment to be more general i.e. "runs on supported platforms"